src/others/contexts/apiMockData.ts

```diff
@@ -1,94 +1,83 @@
-import { Country } from "./api";
+import { Country } from './country';
+import { SERVICE_TYPE } from './service';
 
 export const API_MOCK: Country[] = [
   {
-    id: "ua",
-    name: "Ukraine",
-    info: {
-      services: {
-        inhouse: [
-          {
-            id: "request",
-            name: "Request supplies",
-            type: "supplies",
-            path: "https://requests.ugtf.app",
-          },
-          {
-            id: "psyche",
-            name: "Psychological support",
-            type: "chat",
-            path: "/",
-          },
-        ],
-        external: [
-          {
-            id: "uk_external_1",
```

Comment:
I'm relying on `Service.type` to make the distinction instead, so I don't think having two arrays is neccessary.

Reply:
we could do this via the type but I feel like this is a very implicit, wouldn't you agree?

Reply:
Splitting it into two arrays in the parent, but leaving `type/origin` (`type` for now, but eventually `origin`) on the child means typescript wouldn't complain about this:

```ts
const country: Country = {
  ...
  internal: [{
    ...
    origin: 'External'
  }],
  external: [{
    ...
    origin: 'Internal'
 }]
}
```

There are two potentially conflicting sources of truth. Since we already have `origin/type` on the `Service` interface, I think we should use that one. Perhaps I'm missing your perspective, but I don't find it implicit, it's an array of services, some are marked  `Internal` some `External`.

Reply:
You're right, we don't have to split up the the array into external/internal if we tag each service with an origin.
This is probably easier 👍